controlplane/kubeadm/main.go

```diff
@@ -244,6 +246,12 @@ func setupWebhooks(mgr ctrl.Manager) {
 	if err := (&kcpv1.KubeadmControlPlaneTemplate{}).SetupWebhookWithManager((mgr)); err != nil {
 		setupLog.Error(err, "unable to create webhook", "webhook", "KubeadmControlPlaneTemplate")
 	}
+
+	mgr.GetWebhookServer().Register("/validate-scale-controlplane-cluster-x-k8s-io-v1beta1-kubeadmcontrolplane", &webhook.Admission{
+		Handler: &kcpwebhooks.ScaleValidator{
+			Client: mgr.GetClient(),
+		},
+	})
```

Comment:
@sbueringer Sidenote, the Scale subresource should probably have top-level support in Controller Runtime